torchgeo/models/farseg.py

```diff
@@ -0,0 +1,227 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+"""Foreground-Aware Relation Network (FarSeg) implementations."""
+
+import torch
+import torch.nn as nn
+import torch.nn.functional as F
+from torchvision.models import resnet
+from torchvision.ops import FeaturePyramidNetwork as FPN
+from typing import List, Tuple
+import math
+from collections import OrderedDict
```

Comment:
If you run `isort .` it will resort all of these imports for you. This is required to get the unit tests to pass. Just `pip install isort` if you don't already have it installed. Same for black/flake8/mypy/pydocstyle.